mediaire_toolbox/queue/redis_wq.py

```diff
@@ -94,8 +97,60 @@ def _lease_exists(self, item):
     def put(self, item):
         self._db.lpush(self._main_q_key, item)
 
+    @staticmethod
+    def _get_timestamp(timeunit):
+        if timeunit == 'sec':
+            return time.gmtime(time.time()).tm_sec
+        if timeunit == 'minute':
+            return time.gmtime(time.time()).tm_min
+        if timeunit == 'hour':
+            return time.gmtime(time.time()).tm_hour
+        return RedisWQ._get_timestamp('minute')
+
+    @staticmethod
+    def _get_expirytime(timeunit):
```

Comment:
can i only define 1s, 1min, and 1h as expiry time? not sure if i am getting this right. but i'd rather keep the value flexible and choose a fixed unit (e.g. seconds).

Reply:
Yes, we perhaps should make expiry time also flexible. I was rather thinking of having default expiry times for certain time units.